### DISCUSSION TITLE
Wiki access

### DISCUSSION THREAD

**Discussion by lowlydba** [dataplat/DataSaturdays#73] [Q&A]
Currently it looks as if the wiki is editable by collaborators only - should we open it up to all users, to allow some faster methods of contributing during early stages?

**Comment by way0utwest** [comment]
I am honestly not sure about this. I don't want maintaining the wiki and clearing or reversing misleading items to be a job.

I also think we don't have much in terms of code/tools/process to add as of yet. I would prefer we start to move the toolset forward first.
